python.py

```diff
@@ -0,0 +1,20 @@
+# here is a coment
+
+#
+# block coment
```

Comment:
Typo inside comment in token 'coment'. Possible corrections:comment (62),moment (33),comet (15)